package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "native-template",
-  "version": "13.0.0",
+  "version": "13.0.1",
```

Comment:
This shouldn't be changed manually. it's handled automatically by our release automation. Could you please revert this change?